src/components/TokenDisplay.test.tsx

```diff
@@ -71,6 +69,27 @@ describe('TokenDisplay', () => {
         )
       ).toThrow()
     })
+    it('throws when neither currency nor tokenAddress are supplied', () => {
+      expect(() =>
+        render(
+          <Provider store={store()}>
+            <TokenDisplay showLocalAmount={false} amount={10} testID="test" />
+          </Provider>
+        )
+      ).toThrow()
+    })
```

Comment:
just to confirm, this is the same test as earlier just moved around right?

Reply:
yep, that's correct